src/scene_battle_rpg2k.cpp

```diff
@@ -450,12 +461,12 @@ bool Scene_Battle_Rpg2k::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBase
 			}
 
 			if (action->TargetNext()) {
-				battle_action_state = BattleActionState_Start;
+				battle_action_state = BattleActionState_ConditionHeal;
 				return false;
 			}
 
 			// Reset variables
-			battle_action_state = BattleActionState_Start;
+			battle_action_state = BattleActionState_ConditionHeal;
```

Comment:
You changed all the ```battle_action_state``` transitions. This is **super super risky**, pls don't do this unless you are 100% certain this is required. Because this must be tested with every available battle action available (Normal, different items, different skills, defend) to proof that it doesn't break anything.

One problem I already noticed is, that you introduced some delay before everybodys turn.